docs/source/CppUsage.md

```diff
@@ -78,9 +81,9 @@ If you look in your generated header, you'll see it has
 convenient accessors for all fields, e.g. `hp()`, `mana()`, etc:
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~{.cpp}
-    printf("%d\n", monster->hp());            // `80`
-    printf("%d\n", monster->mana());          // default value of `150`
-    printf("%s\n", monster->name()->c_str()); // "MyMonster"
+    std::cout << "hp : " << monster->hp() << std::endl;            // `80`
+    std::cout << "mana : " << monster->mana() << std::endl;        // default value of `150`
+    std::cout << "name : " << monster->name() << std::endl;        // "MyMonster"
```

Comment:
Same here.. name() returns a `flatbuffers::String` which I am pretty sure ostream doesn't understand. You still need the `c_str()`.

Reply:
Good catch! 
Added c_str() and committed diff.